packages/core/build/config.js

```diff
@@ -103,8 +103,8 @@ const copyConfig = base => {
             from: path.resolve(__dirname, '../src/templates/app/manifest.json'),
             to: 'manifest.json',
             toType: 'file',
-            transform(content, path) {
-                return transformContentUrlBase(content, path, base);
+            transform(content, transform_path) {
```

Comment:
I just changed this because of a warning that was about the `path` key already defined in upper scope.